runtime/oti/j9nonbuilder.h

```diff
@@ -5698,6 +5764,11 @@ typedef struct J9CInterpreterStackFrame {
 	U_8 preservedFPRs[8 * 8]; /* v8-15 */
 	UDATA jitGPRs[32]; /* x0-x31 */
 	U_8 jitFPRs[32 * 8]; /* v0-v31 */
+#elif defined(J9VM_ARCH_RISCV) /* J9VM_ARCH_ARM */
+	UDATA preservedGPRs[13]; /* x2, x8, x9, and x18-x27  */
+	U_8 preservedFPRs[32]; /* f0-f31 */
```

Comment:
Should this be 8 times larger? `U_8 preservedFPRs[32 * 8];`

Reply:
Updated. It might be modified accordingly in JIT based on their real requirements.